### PR TITLE
Create PR template

### PR DESCRIPTION
**Pull request by OrianP**
@rachellimop @isaacojq11 this is a template for pull requests made on this repo. When you make a pull request the PR body in Github will automatically be populated with this content. Because this repo is public it will make it nicer to see history of what was changed if we have more detailed summaries in the pull requests. Is there anything you would like to add to this?

- Added as a hidden file in `.github/`
- Pretty simple and short - @btabram anything else you wanted to include?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### :memo: Summary of changes
+Please summarise changes made in this pull request
+
+### 🤔 Considerations 
+This repository is public. Please ensure:
+- [ ] Template does not include any sensitive information.
```
Comment (btabram) [inline .github/pull_request_template.md:6]:
```suggestion
- [ ] Templates do not include any sensitive information (templates are public)
- [ ] Template versions have been increased if there's entity changes  
```

How about this? I think it'd good to have template versioning as a consideration too

Comment (btabram) [reply .github/pull_request_template.md:6]: Otherwise the template looks good 👍

**Review by isaacojq11 — APPROVED** [review]
Looks good to me! Thanks for making this 😁

**Comment by rachellimop** [comment]
Yeees a great to add it in (and a good reminder for myself!)

**Pull request merged**